sys/Makefile.include

```diff
@@ -77,7 +77,7 @@ ifneq (,$(filter printf_float,$(USEMODULE)))
   endif
 endif
 
-ifneq (,$(filter riotboot_%,$(USEMODULE)))
+ifneq (,$(filter riotboot%,$(FEATURES_USED)))
```

Comment:
I think the ```%``` at the end of riotboot is not necessary. If you feel like, could you confirm and then force-push its removal? I'm OK with merging as is, too. ;)

Reply:
yep!